docs/examples/legacy/Legacy.ipynb

```diff
@@ -0,0 +1,127 @@
+    "\n",
+    "\n",
+    "## Loop\n",
+    "A Loop is the QCoDeS way to acquire one or more arrays of data. Every Loop that’s executed consists of a settable Parameter to be varied, some collection of values to set it to, some actions to do at each setpoint, and some conditions by which to run the Loop.\n",
+    "\n",
+    "An action can be:\n",
+    "- A gettable Parameter (something to measure). Each such Parameter will generate one (or more, if the Parameter itself creates multiple outputs).\n",
+    "- A Task to do (for example you measure once, then have a Task to change a gate voltage, then you measure again, and finally a Task to put the gate voltage back where it was).\n",
+    "- Wait, a specialized task that just delays execution (but may do other things like monitoring the system in that time)\n",
+    "- BreakIf, a callable test for whether to quit (this level of) the Loop.\n",
+    "\n",
+    "Another Loop nested inside the first, with all its own setpoints and actions. Measurements within a nested loop will produce a higher-dimension output array.\n",
+    "\n",
+    "The key loop running conditions are:\n",
+    "- background or not: A background Loop runs in its own separate process, so that you can be doing things in the main process like live plotting, analysis on the data as it arrives, preparing for the next measurement, or even unrelated tasks, while the Loop is running. The disadvantage is complexity, in that you can only use RemoteInstruments, and debugging gets much harder.\n",
+    "- use threads: If true, we will group measurement actions and try to execute them concurrently across several threads. This can dramatically speed up slow measurements involving several instruments, or all instruments are local.\n",
+    "- data manager: If not False, we create another extra process whose job it is to offload data storage, and sync data back to the main process on demand, so that the Loop process can run with as little overhead as possible.\n",
+    "\n",
+    "\n",
+    "### Responsibilities:\n",
+    "- creating the dataset that will be needed to store its data\n",
+    "- where and how to save the data to disk\n",
+    "- generating all the metadata for the DataSet. \n",
+    "> Metadata is intended to describe the system and software configuration to give it context, help reproduce and troubleshoot the experiment, and to aid searching and datamining later. The Loop generates its own metadata, regarding when and how it was run and the Parameters and other actions involved, as well as asking all the Instruments, via a qcodes.station if possible, for their own metadata and including it.\n",
+    "- sequencing actions: the Loop should have the highest priority and the least overhead of extra responsibilities so that setpoints and actions occur with as fast and reliable timing as possible.\n",
```

Comment:
for rendering
```suggestion
    "### Responsibilities:\n",
    "\n",
    "- creating the dataset that will be needed to store its data\n",
    "- where and how to save the data to disk\n",
    "- generating all the metadata for the DataSet. \n",
    "\n",
    "> Metadata is intended to describe the system and software configuration to give it context, help reproduce and troubleshoot the experiment, and to aid searching and datamining later. The Loop generates its own metadata, regarding when and how it was run and the Parameters and other actions involved, as well as asking all the Instruments, via a qcodes.station if possible, for their own metadata and including it.\n",
    "\n",
    "- sequencing actions: the Loop should have the highest priority and the least overhead of extra responsibilities so that setpoints and actions occur with as fast and reliable timing as possible.\n",
```